docs/api/cases/cases-api-get-case-activity.asciidoc

```diff
@@ -0,0 +1,410 @@
+[[cases-api-get-case-activity]]
+== Get case activity API
+++++
+<titleabbrev>Get case activity</titleabbrev>
+++++
+
+Returns all user activity for the specified case.
+
+=== Request
+
+`GET <kibana host>:<port>/api/cases/<case ID>/user_actions` deprecated:[8.1.0, "This endpoint is deprecated and will be removed in a future release."]
```

Comment:
My only concern with having the deprecated warning only on the first endpoint is that a user may think that the `GET <kibana host>:<port>/s/<space_id>/api/cases/<case ID>/user_actions` is not deprecated. For example this https://kibana_128874.docs-preview.app.elstc.co/guide/en/kibana/master/cases-api-get-status.html has a deprecated warning on the top of the page.

Reply:
@cnasikas 
Ah, okay, that was a misunderstanding on my part. Thanks for clarifying. I moved the deprecation message to the top of the page and added it as a block, not an in-line message.

Reply:
Thanks!